content/docs/tech-insights/track-codecov-coverage-results/index.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Track Codecov coverage results
+publishedDate: '2024-07-04T15:15:00.0Z'
+description: How to track Codecov coverage with Roadie Tech Insights
+---
+
+<div style="position: relative; padding-bottom: 56.25%; height: 0;"><iframe src="https://www.loom.com/embed/0cdc0a9fbd3d4ae6b22fb324ca801fe5?sid=aedeab6b-25b4-4d06-bd93-ecd00ba38784" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+
+Tracking the code coverage information for catalog entities is crucial in ensuring the software is secure and compliant. It can be difficult to know which components have coverage activated and are meeting minimum standards.
+
+With Roadie’s Tech Insights feature, there’s an easier way. In this tutorial, we’re going to:
+
+1. Automatically scan Components in the Backstage catalog and record which components have Codecov activated, active, and their % of coverage.
+1. Create a check that can tell us which software is active and activated.
+1. Create a check that can tell us which software is meeting minimum code coverage threshold of 30%.
```

Comment:
the number `30` is a bit low for usual cases. If we can, maybe we could massage it up a bit